_component_design/modal.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Modal
+parent: Content Areas
+layout: component
+category: Components
+usage: Modals to command the user’s attention by obstructing the view of all items not related to the action. Confirmation modals are the most common form and are used when an action could have a destructive result, such as deleting a server or resetting an API key.
+resource: true
+status: unknown
+preview-image: preview-images/modal.svg
+last-modified: 2017-11-01
+---
+
+{% include toc.html %}
+
+## Introduction
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Best practices
+ * Use modals sparingly, only in situations where the user must confirm before continuing, or when the cost of an error could be very high.
```

Comment:
...sparingly and only...
...is very high...

Reply:
👍